src/js/components/award/shared/InfoTooltipContent.jsx

```diff
@@ -1316,6 +1316,25 @@ export const recipientOverviewLoanInfo = (
     </div>
 );
 
+export const stateOverviewLoanInfo = (
```

Comment:
This does not belong in this file considering these tooltips are for `award/shared/`. Probably, `components/state`.

Reply:
Moved it!